### DISCUSSION TITLE
Linking recipes

### DISCUSSION THREAD

**Discussion by thetimelordsofgallifrey** [mealie-recipes/mealie#3612] [Feature Request]
### First Check

- [X] I used the GitHub search to find a similar requests and didn't find it.
- [X] Checked the [tasks tagged](https://github.com/mealie-recipes/mealie/issues?q=is%3Aissue+is%3Aopen+label%3Atask+) issues and verified my feature is not covered

### Please provide a concise description of the problem that would be addressed by this feature.

I have a number of recipes that rely on other recipes, is there any way of linking words in either the ingredients list or instructions to other recipes?

### Please provide a concise description of the feature that would resolve your issue.

Being able to link a main recipe to another recipe, or bring up a box in the middle of the screen

### Please consider and list out some caveats or tradeoffs made in your design decision

It would require a back button in Mealie or might be confusing.

### Additional Information

- [X] If this is accepted I'm willing to submit a PR to provide this feature
- [X] If this is accepted I'm willing to help maintain this feature
- [ ] I'm willing to sponsor/pay a developer to do this work

**Comment by Kuchenpirat** [comment]
Hey, thanks for the feature request. This is one of our most requested features and as such there is already an existing FR for that (see #1761).

Please contribute / upvote to that thread.

**Comment by thetimelordsofgallifrey** [comment]
Hi

Many thanks, I didn’t spot it.

Simon
